test/ptp/tgm/validation/tests/validation.go

```diff
@@ -57,7 +57,7 @@ var _ = Describe("PTP T-GM Validation", func() {
 					fullFirmwareVer := strings.Split(line, ":")[1]
 					By(fmt.Sprintf("NVM firmware version installed: %s", fullFirmwareVer))
 					firmwareVerNum := strings.Split(fullFirmwareVer, " ")[1]
-					Expect(strconv.ParseFloat(firmwareVerNum, 64)).To(BeNumerically(">=", consts.ICEDriverFirmwareVerMinVersion), "linuxptp-daemon is not deployed on cluster")
+					Expect(strconv.ParseFloat(firmwareVerNum, 64)).To(BeNumerically(">=", consts.ICEDriverFirmwareVerMinVersion), "ICE driver version is older than required.")
```

Comment:
The firmware is associated to the card not to the driver. We can change variable name in future PR to maybe something like `NICFirmwareVerMinVersion`?

```suggestion
Expect(strconv.ParseFloat(firmwareVerNum, 64)).To(BeNumerically(">=", consts.ICEDriverFirmwareVerMinVersion), "NVM Firmware version is older than expected.")
```